Contributions.md

```diff
@@ -0,0 +1,50 @@
+# Adding Contributions
+
+If you have an example you think would benefit users of the CCR community, you can follow these steps to create a fork of this repository and submit a pull request for CCR staff to review:
+
+1. [Fork this repo](https://github.com/ubccr/ccr-examples/fork)
+
+2. Clone your fork and enter it's directory:
+
+    ```bash
+    git clone https://github.com/[YourUser]/ccr-examples.git
+    cd ccr-examples
+    ```
+
+3. Start adding to or updating the repository!
+
+4. Submit a [pull request](https://github.com/ubccr/ccr-examples/pulls)
+
+## Standards and Guidelines
+
+Keep examples organized in respective per example directories following the established [directory structure](README.md#navigating-the-repository) and remember to apply any necessary changes to existing files (e.g., table of contents). Please do your best to align the format of your submission with the following standards:
+
+### README Files
+
+- Include a markdown `README.md` file with clear and concise instructions on how to modify and reproduce the example.
+- Separate your `README` into sections and clearly ennumerate steps required to reproduce the example.
+- Avoid repitition by providing links to information already detailed in the [CCR documentation](https://docs.ccr.buffalo.edu/en/latest/) or other `README` files in this repository.
```

Comment:
```suggestion
- Avoid repetition by providing links to information already detailed in the [CCR documentation](https://docs.ccr.buffalo.edu/en/latest/) or other `README` files in this repository.
```

Reply:
Based on discussion in team meeting, add a few examples here of what we mean by things not to include: how to start a job, how to login to a compute node, etc